website/docs/script/js/js-quick-start.mdx

```diff
@@ -0,0 +1,279 @@
+---
+id: js-quick-start
+title: "Quick Start"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Link from "@docusaurus/Link";
+import TutorialHeader from "@components/TutorialHeader";
+import Tooltip from "@components/Tooltip";
+import ScriptTools from "./../_ScriptTools.mdx";
+
+# JavaScript Quick Start
+
+<TutorialHeader
+  time={"5 - 7 min"}
+  customTools={[
+    <div key="ckb-js-vm">
+      <Link
+        to="/docs/getting-started/installation-guide#6-ckb-js-vm"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        ckb-js-vm
+      </Link>
+    </div>,
+    <div key="offckb">
+      <Link
+        to="docs/sdk-and-devtool/offckb"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        】=offckb {">= 0.3.2"}
```

Comment:
offckb is not used here. instead, there is a missing required tool which is ckb-debugger